src/lib/utils/node_config.ts

```diff
@@ -0,0 +1,52 @@
+
+import { Noise } from '@chainsafe/libp2p-noise';
+import { Mplex } from '@libp2p/mplex';
+import { Bootstrap } from '@libp2p/bootstrap';
+import { KadDHT } from '@libp2p/kad-dht';
+import { WebSockets } from '@libp2p/websockets';
+import { WebRTCStar } from '@libp2p/webrtc-star';
+
+import { FloodSub } from '@libp2p/floodsub';
+
+const dht = new KadDHT({
+  protocolPrefix: '/archaeologist-service',
+  clientMode: false
+});
+
+const webRtcStar = new WebRTCStar();
+
+export const nodeConfig = {
+  addresses: {
+    // Add the signaling server address, along with our PeerId to our multiaddrs list
+    // libp2p will automatically attempt to dial to the signaling server so that it can
+    // receive inbound connections from other peers
+    listen: process.env.REACT_APP_SIGNAL_SERVER_LIST!.split(',').map((s: string) => s.trim()).map((server: any) => {
+      return `/dns4/${server}/tcp/443/wss/p2p-webrtc-star`;
+    })
+  },
+  transports: [
+    new WebSockets(),
```

Comment:
Does the behavior of this browser node change if this `WebSockets` transport is removed?  

I'm wondering if after the peer is discovered via `webRtcStar`, if it continues to communicate over that transport and if we even need `WebSockets`. It may be that we do, but wanted to verify.

Reply:
From my tests, it doesn't seem to change anything when `WebSockets` is removed